roles/contiv_network/templates/netplugin.j2

```diff
@@ -0,0 +1,7 @@
+{# If an interface is specified for netplugin, use it #}
+{# Otherwise, hope for the best                       #}
```

Comment:
While I agree with fixing broken things, I don't think it needs to be part of this effort. I believe in fixing things incrementally.